examples/nodejs/FuzzTestCase.fuzz.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { FuzzedDataProvider } = require("@jazzer.js/core");
+const exploreMe = require("./ExploreMe");
+
+describe("My describe", () => {
+    it.fuzz("My fuzz test", (jazzerBuffer) => {
```

Comment:
```suggestion
    test.fuzz("My fuzz test", jazzerBuffer => {
```
`it` is usually used in jest to form a sentence like `it("should be return the square root of a", ()=>{ ... })` 
As there is no expected outcome when running fuzz tests we can just use `test.fuzz` which seems a better fit semantically 

Also the parentheses on `(jazzerBuffer)` are not necessary with only one parameter